### PR TITLE
route employ added

### PR DESCRIPTION
**Pull request by donovankg**
added in the employee table with mock data.

### CODE REVIEW

File: client/app/employee/employee.controller.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
+'use strict';
+(function(Employee) {
+
+  angular.module('maerkApp.employee')
+    .controller('EmployeeController', function($scope) {
+      var errors = {};
+      var submitted = false;
+
+    //  console.log(Employee);
+    $scope.nutritionList = [
```
Comment (cleverjam) [inline client/app/employee/employee.controller.js]: Make sure you erase what you are not using

File: client/app/employee/employee.controller.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
+'use strict';
+(function(Employee) {
+
+  angular.module('maerkApp.employee')
+    .controller('EmployeeController', function($scope) {
```
Comment (cleverjam) [inline client/app/employee/employee.controller.js]: Use 'this' instead of $scope

File: client/app/employee/employee.router.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular.module('maerkApp.employee')
+  .config(function($stateProvider) {
+    $stateProvider.state('employee', {
```
Comment (cleverjam) [inline client/app/employee/employee.router.js]: is this a "sub state" of main?? main.employee??

File: client/app/main/main.js [hunk 4]
```diff
@@ -6,5 +6,12 @@ angular.module('maerkApp')
       url: '/main',
       template: '<main class ="flex layout-row"></main>',
       authenticate: true
+    })
+    .state('main.employee', {
```
Comment (cleverjam) [inline client/app/main/main.js]: main is suppose to be abstract

File: client/app/employee/employee.controller.js (hunk 2 above)
Comment (donovankg) [inline client/app/employee/employee.controller.js]: this doesn't work when you replace it for $scope

File: client/app/employee/employee.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                          'first_name',
+                          'last_name',
+                          'client',
+                          'skill',
+                          'recruiter',
+                          'salary'
+                            ]
+                      }">
+  <mdt-header-row>
+        <mdt-column align-rule="right">first name</mdt-column>
```
Comment (cleverjam) [inline client/app/employee/employee.html]: emp.employess   (after changing from $scope to "this" inside the controller)

File: client/app/main/main.js (hunk 4 above)
Comment (lchapoy) [inline client/app/main/main.js]: If main route is not abstract you will have an error when reloading employee routes

File: client/app/employee/employee.controller.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
```
Comment (cleverjam) [inline client/app/employee/employee.controller.js]: remove dependency for $scope if you are not using any more

File: client/app/main/main.js (hunk 4 above)
Comment (donovankg) [inline client/app/main/main.js]: this issue is fixed

**Pull request merged**